### PR TITLE
Adding Gateway API v0.8.0 Blog Post

### PR DESCRIPTION
**Pull request by robscott**
This is intended to line up with the Gateway v0.8.0 release that's currently scheduled for Tuesday, August 29. I've mostly copied @kflynn's post from https://github.com/kubernetes-sigs/gateway-api/pull/2181, but have rearranged some things and added other non-mesh content, hopefully that makes sense, feedback is very appreciated.

/cc @kflynn @howardjohn @keithmattix @mikemorris @michaelbeaumont @shaneutt @youngnick @sftim 
/hold until release

### CODE REVIEW

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [inline content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:8]: @kflynn @howardjohn @keithmattix @mikemorris @michaelbeaumont please let me know if I made any mistakes here

**Comment by sftim** [comment]
On behalf of the website blog team, I'll flag this to Release Comms as we're still running the post-release blogs for v1.28 of Kubernetes.

**Comment by robscott** [comment]
@sftim good to know, thanks! I was hoping we'd come in just after 1.28 release posts, but let us know if we need to adjust our date.

**Comment by youngnick** [comment]
LGTM, nice work everyone!

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (sftim) [inline content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:176]: These relative links don't work. They need to change to be https://gateway-api.sigs.k8s.io/ absolute links, and the https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/ link should switch to a relative link.

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (frankbu) [inline content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:100]: Would be good to have a link here?
```suggestion
This release marks the beginning of a transition from webhook validation to
[CEL validation][cel] that is built into the CRDs. That will mean different things
```

**Review by howardjohn — COMMENTED** [review]
LGTM

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (sftim) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:166]: Doesn't look right to me. See https://github.com/kubernetes/website/pull/42727/files#r1307737224

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (kflynn) [inline content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:20]:
```suggestion
## Service mesh support in the Gateway API
```

**Review by kflynn — CHANGES_REQUESTED** [review]
Thanks @robscott! I made one more change after you grabbed this, see my comment inline. 🙂

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (kflynn) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Great idea. 🙂

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (kflynn) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Oh ouch, you're right.

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (kflynn) [inline content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:186]:
```suggestion
[cel]:/docs/reference/using-api/cel/
[crd]:/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
```

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Yep, sorry, that's on me. Should be updated across the board now and have manually verified each link. They all should work with the exception of the release link which will work once a release is published.

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:18]: I think this should be covered now that I've copied everything from your PR back over here, let me know if I missed anything.

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (mikemorris) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Lowercasing `independent` as suggested above then LGTM

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (mikemorris) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md:18]: I do like the more direct section heading suggestion better, think that's the last update!

**Review by mikemorris — APPROVED** [review]
/lgtm

**Review by keithmattix — APPROVED** [review]
/approve

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Looks like my last push missed some things, updated, thanks!

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Updated, thanks!

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: @sftim I don't think that link works for me, the CEL link has been added and appears to work though.

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Updated, thanks!

File: content/en/blog/_posts/2023-08-29-Gateway-API-v080.md
Comment (robscott) [reply content/en/blog/_posts/2023-08-29-Gateway-API-v080.md]: Updated, thanks!

**Comment by robscott** [comment]
🎉 [Gateway API v0.8.0 is live](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.8.0), this can go in as soon as it's approved.

/hold cancel

**Comment by sftim** [comment]
/lgtm
/approve

Thanks

**Pull request merged**
